scripts/deploy.js

```diff
@@ -28,7 +29,7 @@ async function main() {
     console.log(`Using network ${chalk.bold(hre.network.name)} (${chalk.bold(hre.network.config.chainId)})`);
 
     console.log(`> Getting signers to deploy RenPool contract`);
-    const [owner] = await ethers.getSigners();
+    const owner = new ethers.Wallet(accounts[0], ethers.provider);
```

Comment:
Is there a reason to use `Wallet` here explicitly? Why can't we use `getSigners`?

I would like to use this script to deploy on both local (when invoked from `hardhat console`) and remote networks. I can't use `Wallet` here since `accounts` is not defined for local network in our hardhat config. But `getSigners` will work for both, since it will return the account for remote networks.

Reply:
Ok, if that's the case, we can then use getSigners. For some reason last time i tried it was loading the wrong wallet, but probably i made some mistake